include/mppic/parameters_handler.hpp

```diff
@@ -65,11 +65,15 @@ class ParametersHandler
   template<typename T>
   static auto as(const rclcpp::Parameter & parameter);
 
+private:
   std::mutex parameters_change_mutex_;
   rclcpp::Logger logger_{rclcpp::get_logger("MPPIController")};
   rclcpp::node_interfaces::OnSetParametersCallbackHandle::SharedPtr
     on_set_param_handler_;
   rclcpp_lifecycle::LifecycleNode::WeakPtr node_;
+  std::string node_name_;
+
+  bool verbose_{false};
```

Comment:
This doesn't appear to ever be overridable / changable

Reply:
add usage